app/javascript/app/constants/links.js

```diff
@@ -1,3 +1,3 @@
 export const ENVIRONMENTAL_AFFAIRS_URL = 'https://www.environment.gov.za/';
-
+export const GHG_NATIONAL_REPORT = 'https://unfccc.int/sites/default/files/resource/28147395_South%20Africa-BUR2-1-South%20Africa_%20GHG%20Inventory%20Report%20-%202000-2012.pdf';
 export default { ENVIRONMENTAL_AFFAIRS_URL };
```

Comment:
I think we can remove the `export default ...`

Reply:
This actually comes from an eslint rule: https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/prefer-default-export.md
I think that maybe is not needed because we keep forgetting to update this export defaults, so maybe we can remove it from the es-lint rules

Reply:
+1, I think it creates confusion

Reply:
Sorry, I was wrong about this rule. It only applies when we have one export. It seems we have been using lots of unnecessary default exports 😅 I will remove each time I update a file. This one is fixed